.github/workflows/runtime-sync.yml

```diff
@@ -43,10 +43,25 @@ jobs:
         name: results
         path: artifacts
     - name: Check and Notify
+      id: check
       shell: pwsh
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         # Test this script using an issue in the local forked repo
         $issue = 'https://api.github.com/repos/dotnet/aspnetcore/issues/18943'
-        .\aspnetcore\.github\workflows\ReportDiff.ps1
\ No newline at end of file
+        $changed = .\aspnetcore\.github\workflows\ReportDiff.ps1
+        echo "::set-output name=changed::$changed"
+    - name: Send PR
+      if: steps.check.outputs.changed == 'true'
+      # https://github.com/marketplace/actions/create-pull-request
+      uses: peter-evans/create-pull-request@v2
```

Comment:
Any security issue with depending on this?

Reply:
Everything it has read access to is already public, and all the branches we care about have branch protection. I expect the worst it could do is spam our issues/PRs.